identity/manager.go

```diff
@@ -95,13 +100,99 @@ func (m *Manager) Create(ctx context.Context, i *Identity, opts ...ManagerOption
 	}
 
 	if err := m.r.PrivilegedIdentityPool().CreateIdentity(ctx, i); err != nil {
+		if errors.Is(err, sqlcon.ErrUniqueViolation) {
+			return m.generateCredentialHint(ctx, err, i)
```

Comment:
Is this behavior something we should make configurable? It is revealing quite a lot of information, so not sure if this should be enabled by default.